src/Schema/Table.php

```diff
@@ -346,6 +349,44 @@ public function addColumn($name, $typeName, array $options = [])
         return $column;
     }
 
+    /** @return array<string, string> */
+    final public function getRenamedColumns(): array
+    {
+        return $this->renamedColumns;
+    }
+
+    /**
+     * @throws Exception
+     * @throws SchemaException
+     */
+    final public function renameColumn(string $oldName, string $newName): Column
+    {
+        $oldName = $this->normalizeIdentifier($oldName);
+        $newName = $this->normalizeIdentifier($newName);
+
+        if ($oldName === $newName) {
+            throw new Exception("You are trying to rename the column ${oldName} but you didn't change it's name");
```

Comment:
```suggestion
            throw new Exception(sprintf(
                'You are trying to rename the column %s but you didn\'t change it\'s name',
                $oldName,
            ));
```

The `${foo}` syntax is deprecated and we don't allow string interpolation in this repository.